rviz_plugin_tutorials/src/drive_widget.cpp

```diff
@@ -79,69 +78,67 @@ void DriveWidget::paintEvent( QPaintEvent* event )
   int w = width();
   int h = height();
   int size = (( w > h ) ? h : w) - 1;
-  int hpad = ( w - size ) / 2;
-  int vpad = ( h - size ) / 2;
+  int hpad = (w - size) / 2;
+  int vpad = (h - size) / 2;
 
-  QPainter painter( this );
-  painter.setBrush( background );
-  painter.setPen( crosshair );
+  QPainter painter(this);
+  painter.setBrush(background);
+  painter.setPen(crosshair);
 
   // Draw the background square.
-  painter.drawRect( QRect( hpad, vpad, size, size ));
+  painter.drawRect(QRect(hpad, vpad, size, size));
 
   // Draw a cross-hair inside the square.
-  painter.drawLine( hpad, height() / 2, hpad + size, height() / 2 );
-  painter.drawLine( width() / 2, vpad, width() / 2, vpad + size );
+  painter.drawLine(hpad, height() / 2, hpad + size, height() / 2);
+  painter.drawLine(width() / 2, vpad, width() / 2, vpad + size);
 
   // If the widget is enabled and the velocities are not zero, draw
   // some sweet green arrows showing possible paths that the wheels of
   // a diff-drive robot would take if it stayed at these velocities.
-  if( isEnabled() && (angular_velocity_ != 0 || linear_velocity_ != 0 ))
-  {
+  if (isEnabled() && (angular_velocity_ != 0 || linear_velocity_ != 0)) {
     QPen arrow;
-    arrow.setWidth( size/20 );
-    arrow.setColor( Qt::green );
-    arrow.setCapStyle( Qt::RoundCap );
-    arrow.setJoinStyle( Qt::RoundJoin );
-    painter.setPen( arrow );
+    arrow.setWidth(size / 20);
+    arrow.setColor(Qt::green);
+    arrow.setCapStyle(Qt::RoundCap);
+    arrow.setJoinStyle(Qt::RoundJoin);
+    painter.setPen(arrow);
 
     // This code steps along a central arc defined by the linear and
     // angular velocites.  At each step, it computes where the left
     // and right wheels would be and collects the resulting points
     // in the left_track and right_track arrays.
-    const int step_count = 100;
-    QPointF left_track[ step_count ];
-    QPointF right_track[ step_count ];
-
-    float half_track_width = size/4.0;
-
-    float cx = w/2;
-    float cy = h/2;
-    left_track[ 0 ].setX( cx - half_track_width );
-    left_track[ 0 ].setY( cy );
-    right_track[ 0 ].setX( cx + half_track_width );
-    right_track[ 0 ].setY( cy );
-    float angle = M_PI/2;
-    float delta_angle = angular_velocity_ / step_count;
-    float step_dist = linear_velocity_ * size/2 / linear_scale_ / step_count;
-    for( int step = 1; step < step_count; step++ )
-    {
+    const int kStepCount{100};
+    QPointF left_track[kStepCount];
+    QPointF right_track[kStepCount];
+
+    float half_track_width = size / 4.0;
```

Comment:
Better to avoid implicit conversion. I expect that macOS and/or Windows CI will warn about this.

```suggestion
    float half_track_width = static_cast<float>(size) / 4.0f;
```

Reply:
Thanks, fixed all instances of this